include/mbgl/style/source.hpp

```diff
@@ -75,6 +75,15 @@ class Source : public mbgl::util::noncopyable {
     virtual void loadDescription(FileSource&) = 0;
     void setPrefetchZoomDelta(optional<uint8_t> delta) noexcept;
     optional<uint8_t> getPrefetchZoomDelta() const noexcept;
+
+    // Sets a limit for how much parent tile can be overscaled.
+    //
+    // When a set of tiles for a current zoom level is being rendered and some of the
+    // ideal tiles that cover the screen are not yet loaded, parent tile could be
+    // used instead. This might introduce unwanted rendering side-effects, especially
+    // for raster tiles that are overscaled multiple times.
+    void setMaxOverscaleFactorForParentTiles(optional<uint8_t> overscaleFactor) noexcept;
```

Comment:
`setTilesOverscaleFactorLimit` ? should `Parent` be necessary the part of the name?

Reply:
Initially I had a name without `Parent` postfix, but I found it confusing. Developers might think that we set an overscale limit for all tiles. I know that name is a bit long, however, it is disambiguous.